_config.yml

```diff
@@ -7,19 +7,21 @@
 
 # Site settings
 title: Iridium Browser
-email: iridium@lists.inai.de
+email: wg-iridium@osb-alliance.com
 description: > # this means to ignore newlines until "baseurl:"
   The Iridium Browser is based on the Chromium code base. All modifications enhance the privacy of the user and make sure the most secure technologies are used.
 hotpic: /images/hotpics/Iridium-hotpic.png
 hotpic-fb: /images/hotpics/Iridium-hotpic_FB.png
 hotpic-tw: /images/hotpics/Iridium-hotpic_TW.png  
-keywords: "Iridium Browser, Iridium, Browser, Privacy, Security, Open Source, Secure Browser Project, Secure Browser, Privacy Browser, Chromium based, Windows, MacOS, Ubuntu, Debian, Mint, openSUSE, Fedora, Linux, Red Hat Enterprise Linux"
+keywords: "Iridium Browser, Iridium, Browser, Privacy, Security, Open Source, Secure Browser Project, Secure Browser, Privacy Browser, Chromium based, Windows, macOS, Ubuntu, Debian, Mint, openSUSE, Fedora, Linux, Red Hat Enterprise Linux, CentOS"
   
 baseurl: "" # the subpath of your site, e.g. /blog
-url: https://iridiumbrowser.de # the base hostname & protocol for your site
+url: http://iridium.souldanger.com/ # the base hostname & protocol for your site
```

Comment:
This seems wrong!

Reply:
Thanks for spotting this... it was late last night testing...!
FIXED!